src/lag/nas_int_lag_api.cpp

```diff
@@ -511,44 +511,6 @@ t_std_error nas_lag_set_admin_status(hal_ifindex_t index, bool enable)
     return STD_ERR_OK;
 }
 
-t_std_error nas_lag_get_admin_status(hal_ifindex_t index, bool &admin_status)
```

Comment:
Why are we removing these functions? Might it be useful to applications to be able to query the admin/oper status of a LAG? Or are these now being accomplished via other means not shown in the diff?

Reply:
We don't need them. admin/oper status is already properly updated. I made these functions but never use them so now I'm getting rid of them.

Reply:
Ok cool.